### ISSUE TITLE
Build system should check for 'lit' installed command, not 'lit.py'

### ISSUE THREAD

**Issue by Apteryks** [root-project/cling#432]
### Describe the bug
The build system currently looks (as a fallback) for an installed `lit.py` command.  The PyPI available `lit` package rather installs a `lit` command.

### Expected behavior
`lit` should be found.

### Actual behavior
`lit` is not found, test suite is not run.

### Setup
<!--
1. Cling version 0.9
2. Operating system Guix System
3. How you obtained Cling: from sources

**Comment by Apteryks** [comment]
It should probably check for these 3 things:
`lit`,  `llvm-lit` or `llvm-lit.py`.

When using LLVM as a system library, the AddLLVM CMake module defines its own LLVM lit file name as llvm-lit or llvm-lit.py on Windows.

It can be user specified via the `LLVM_EXTERNAL_LIT` CMake variable, which takes the full path of your `lit` command.

**Comment by Axel-Naumann** [comment]
Any chance that you can propose a patch?

**Comment by ferdymercury** [comment]
You're right, I saw several references to lit inside cpt.py, but I see they are outside too, so sorry for the noise.
